File: Challenge/FIZZBUZZ/fizzbuzz.py
def fizzbuzz(x):
    """Simple FizzBuzz implementation."""

    if x % 3 == 0 and x % 5 == 0:
        return "FizzBuzz"
    elif x % 3 == 0:
        return "Fizz"
    elif x % 5 == 0:
        return "Buzz"
    else:
        return str(x)

if __name__ == '__main__':
    sequence = '\n'.join(fizzbuzz(x) for x in range(1, 100))
    print(sequence)

File: Challenge/FIZZBUZZ/fizzbuzz.kt
fun main(){
  fizzbuzz(
    start = 1,
    end = 100
  )
}


fun fizzbuzz(start: Int, end: Int) { 

    for (i in start..end) { // loop from start to end
        if (i % 3 == 0 && i % 5 == 0) println("fizzbuzz") // if divisible by 3 and 5
        else if (i % 3 == 0) println("fizz") // if divisible by 3
        else if (i % 5 == 0) println("buzz") // if divisible by 5
        else println(i) // else print the number
    }
}